Neos.Neos/Resources/Public/JavaScript/Services/Configuration.js

```diff
@@ -29,7 +29,7 @@ const init = () => {
   }
 
   // deprecated - to be removed in 8.0
```

Comment:
Indeed, we just need to remove the compatibility part so that the window.Typo3Neos is not working anymore.
And if packages just don't want to change that because they need to support more versions at one, they can check that own their own and copy the API stuff.